flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
for ther record it's disappointing to not build for aarch64.

Reply:
Hmm, it looks like the game engine recently added support for ARM Linux. I should be able to provide this.
I'll do some testing with aarch64 and if it works out I'll provide an aarch64 build tonight.

Reply:
I've now compiled a new build with a more recent version of the game engine and bundled the aarch64 libraries.
flathub.json has been updated accordingly.
I believe this should work on an aarch64 platform, but I don't have a device on which to test. I'll either buy a raspberry pi 4 or keep trying to get a working arm64 vm.